lib/Lib.re

```diff
@@ -206,4 +213,24 @@ let duneEject = (projectPath, manifestFile, subpackageNameOrPath) => {
     );
     write(Path.(pesyPackage.pkg_path / "dune"), DuneFile.toString(duneFile));
   };
+};
+
+let validateManifestFile = (projectPath, manifestFile) => {
+  let conf = PesyConf.get(manifestFile);
+  let pkgs = PesyConf.pkgs(conf);
+
+  let duneProjectPath = Path.(projectPath / "dune-project");
+  let duneVersion =
+    DuneFile.ofFile(duneProjectPath) |> DuneProject.findLangVersion;
+
+  let rootName = PesyConf.rootName(conf);
+  let _ =
+    try(
+      pkgs
+      |> List.map(PesyConf.toPesyConf(projectPath, rootName, ~duneVersion))
+    ) {
+    | x => raise(x)
+    };
```

Comment:
@ManasJayanth for validation I chose to re-use the `toPesyConf` function instead of 
maintaining separate validation function is this okay?

Reply:
No problem!